src/Keboola/StorageApi/Downloader/BlobStorageRetryMiddleware.php

```diff
@@ -19,6 +21,14 @@ class BlobStorageRetryMiddleware
     const DEFAULT_NUMBER_OF_RETRIES = 5;
     const DEFAULT_RETRY_INTERVAL = Resources::DEFAULT_RETRY_INTERVAL;
 
+    // Only these classes are retried, not subclasses.
+    // ServerException and ClientException are subclassed of the RequestException,
+    // but they are covered by retried HTTP codes, see generalRetryDecider method.
+    const RETRIED_EXCEPTIONS = [
+        RequestException::class, // eg. cURL error 56: OpenSSL SSL_read:
```

Comment:
Retry middleware doteraz fungovalo iba na HTTP kodoch, no potrebujeme retry aj na chyby spojenia.
Klient si obnovit projekt cez `project-restore` komponentu, a spadne mu to po X hodinach na sietovu chybu v Azure.